### PR TITLE
Better emphasis on developing and using the stackclub package

### PR DESCRIPTION
**Pull request by drphilmarshall**
Turns out I had already added some notes to the Getting Started file, but I think the emphasis was a bit off - you're right that everyone should set up the package, just so they can use it.

### CODE REVIEW

File: GettingStarted/GettingStarted.md [hunk 1]
```diff
@@ -61,18 +61,20 @@ Broadly useful, small datasets are available in `/project/shared/data`  - this i
...
-This will put the `stackclub` folder on your path. You may find the following lines useful to add to your notebook as you develop the library:
+This will put the repo's `stackclub` folder on your path. When developing the package, you may find it useful to add the following lines to your notebook:
 ```python
 %load_ext autoreload
 %autoreload 2
 ```
-This enables you to repeatedly `import stackclub` as you update the library code.
+This enables you to repeatedly `import stackclub` as you update the library code. The above lines are in the [template notebook](templates/template_Notebook.ipynb), for your convenience.
+
+If you are not developing this package, you can install it using pip, like this:
```
Comment (kadrlica) [inline GettingStarted/GettingStarted.md]: This will only work if you have permission to write to your base python site-packages.

**Comment by kadrlica** [comment]
There is also a permission issue with the pip install directions in [FindingDocs.ipynb](https://github.com/LSSTScienceCollaborations/StackClub/blob/master/GettingStarted/FindingDocs.ipynb).

**Comment by kadrlica** [comment]
There is also an issue with this instruction

    If you are developing the stackclub package (eg by adding modules to it to support 
    the Stack Club tutorial that you are writing, you'll need to make a local, editable installation. 
    In the top level folder of the StackClub repo, do:

    In [ ]: ! cd .. && python setup.py -q develop --user && cd -

If you are actually in the top level of the StackClub repo, I think that you don't need the `cd ..`

**Comment by drphilmarshall** [comment]
Thanks @kadrlica - I think I fixed the problems you identified.

**Pull request merged**